lib/cdo/http_cache.rb

```diff
@@ -86,7 +86,6 @@ class HttpCache
     starwarsblocks
     mc
     frozen
-    gumball
```

Comment:
Removed the [gumball](https://studio.code.org/s/gumball) tutorial to keep the list of cache behaviors under 60, as per discussion here: https://codedotorg.slack.com/archives/CFGAVL2CA/p1731085082954289